src/payment/strategies/humm/humm-payment-strategy.ts

```diff
@@ -0,0 +1,59 @@
+import { FormPoster } from '@bigcommerce/form-poster';
+
+import { CheckoutStore,  InternalCheckoutSelectors } from '../../../checkout';
+import { OrderActionCreator, OrderRequestBody } from '../../../order';
+import { OrderFinalizationNotRequiredError } from '../../../order/errors';
+import { PaymentArgumentInvalidError } from '../../errors';
+import PaymentActionCreator from '../../payment-action-creator';
+import { PaymentRequestOptions } from '../../payment-request-options';
+import { ExternalPaymentStrategy } from '../external';
+
+export default class HummPaymentStrategy extends ExternalPaymentStrategy {
+
+    constructor(
```

Comment:
you do not need to do this if there is no custom logic in the implementation of the constructor